tests/db/test_bigchain_api.py

```diff
@@ -185,11 +185,11 @@ def test_get_previous_block_id(self, b):
 
     def test_create_new_block(self, b):
         new_block = b.create_block([])
-        block_hash = hash_data(util.serialize(new_block['block']))
+        block_hash = crypto.hash_data(util.serialize(new_block['block']))
 
         assert new_block['block']['voters'] == [b.me]
         assert new_block['block']['node_pubkey'] == b.me
-        assert PublicKey(b.me).verify(util.serialize(new_block['block']), new_block['signature']) is True
+        assert crypto.VerifyingKey(b.me).verify(util.serialize(new_block['block']), new_block['signature']) is True
```

Comment:
If `VerifyingKey` is used to verify a signature, why it's not just a function? Why do we need an object here?